src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -388,19 +389,66 @@ private function addRevealMenuItem(ItemInterface $menu): void
     /**
      * @param \Knp\Menu\ItemInterface $menu
      */
-    private function addHideMenuItem(ItemInterface $menu): void
+    private function addHideMenuItem(ItemInterface $menu, bool $canHide): void
     {
+        $attributes = [
+            'data-actions' => 'hide',
+            'class' => 'ez-btn--hide',
+        ];
+
         $menu->addChild(
             $this->createMenuItem(
                 self::ITEM__HIDE,
                 [
                     'extras' => ['icon' => 'hide'],
-                    'attributes' => [
-                        'data-actions' => 'hide',
-                        'class' => 'ez-btn--hide',
-                    ],
+                    'attributes' => $canHide
+                        ? $attributes
+                        : array_merge($attributes, ['disabled' => 'disabled']),
                 ]
             )
         );
     }
+
+    private function canCreateAnywhere(Content $content): bool
```

Comment:
Just a thought as I know this kinda dosent create much problem here, but should we not write method like this in opposite way i.e. returning `false` by default and handle the cases when it should return true?

Reply:
I don't see a problem to properly modify the functions for this, which may allow us to avoid problems in the future if we add new limitations and forget to add them here